src/components/progress-tracker/progress-tracker.style.ts

```diff
@@ -59,12 +59,14 @@ const StyledProgressBar = styled.span<
     position: relative;
     background-color: var(--colorsSemanticNeutral200);
     border: 1px solid ${getBorderColour({ progress, error })};
-    border-radius: ${theme.roundedCornersOptOut
-      ? "25px"
-      : "var(--borderRadius400)"};
+    border-radius: ${
+      theme.roundedCornersOptOut ? "25px" : "var(--borderRadius400)"
+    };
     overflow-x: hidden;
-    height: ${getHeight(size)};
-    width: 100%;
+    width: 100%
+    // We are adding 2px here to account for the lost height due to the box-sizing.
+    height: ${`${getHeight(size)} + 2px`};
```

Comment:
**suggestion**: given the `InnerBar` has the same height, what do you think about the following? 

```suggestion
    min-height: fit-content;
```

This way we wouldn't need to hardcode the height and it should automatically resize itself to fit the inner bar.